CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 #### Added
 - Add `useBaseInternationalization` to Project Spec Options to opt out of Base Internationalization. [#961](https://github.com/yonaskolb/XcodeGen/pull/961) @liamnichols
+- Add `Scheme.StoreKitConfiguration` to allow specifying StoreKit Configuration in Scheme and TargetScheme, supporting either xcodeproj and xcworkspace. [#964](https://github.com/yonaskolb/XcodeGen/pull/964) @jcolicchio
```

Comment:
```suggestion
- Add `Scheme.StoreKitConfiguration` to allow specifying StoreKit Configuration in Scheme and TargetScheme, supporting either xcodeproj or xcworkspace. [#964](https://github.com/yonaskolb/XcodeGen/pull/964) @jcolicchio
```

Reply:
I can make this change, but I think it's an important distinction that a scheme can support one or the other, but not both simultaneously

Reply:
Oops I'm dumb, I read the diff backwards 😨